exasol/ds/sandbox/lib/asset_printing/print_assets.py

```diff
@@ -1,33 +1,47 @@
 import fnmatch
+import humanfriendly
 import urllib.parse
-from typing import Tuple, Optional, List, Dict
 
-import humanfriendly
+from datetime import datetime
+from dataclasses import dataclass
+from inspect import cleandoc
+from typing import Dict, List, Optional, TextIO, Tuple
 
 from exasol.ds.sandbox.lib.asset_id import AssetId
 from exasol.ds.sandbox.lib.asset_printing.mark_down_printer import MarkdownPrintingFactory
-from exasol.ds.sandbox.lib.asset_printing.printing_factory import PrintingFactory, TextObject, \
-    HighlightedTextObject
+from exasol.ds.sandbox.lib.asset_printing.printing_factory import (
+    PrintingFactory,
+    CodeBlockTextObject,
+    HighlightedTextObject,
+    TextObject,
+    TitleTextObject,
+)
 from exasol.ds.sandbox.lib.asset_printing.rich_console_printer import RichConsolePrintingFactory
 from exasol.ds.sandbox.lib.aws_access.aws_access import AwsAccess
 from enum import Enum
 
 from exasol.ds.sandbox.lib.aws_access.cloudformation_stack import CloudformationStack
 from exasol.ds.sandbox.lib.tags import DEFAULT_TAG_KEY
 from exasol.ds.sandbox.lib.vm_bucket.vm_dss_bucket import find_vm_bucket, find_url_for_bucket
+from exasol.ds.sandbox.lib.dss_docker import DEFAULT_ORG_AND_REPOSITORY
 
 
 class AssetTypes(Enum):
+    DOCKER = "docker"
     AMI = "ami"
-    VM_S3 = "s3-object"
     SNAPSHOT = "snapshot"
     EXPORT_IMAGE_TASK = "export-image-task"
+    VM_S3 = "s3-object"
     CLOUDFORMATION = "cloudformation"
     EC2_KEY_PAIR = "ec2-key-pair"
 
+    @staticmethod
+    def from_name(name: str):
+        return next(a for a in AssetTypes if a.value == name)
```

Comment:
Check if name is found and throw an exception with a helpful message

Reply:
I will use a dedicated exception and add tests.